src/internal_modules/roc_audio/latency_tuner.cpp

```diff
@@ -422,18 +562,178 @@ void LatencyTuner::report_() {
+                    (long)target_latency_, (double)cur_tl_ns / core::Millisecond,
+                    (long)new_tl_ts, (double)new_tl_ns / core::Millisecond);
+            roc_log(LogDebug,
+                    "Latency monitor:"
+                    "\testimate %.3fms * %.3f = %.3fms,\tnew tl  %.3fms * %f = %.3fms",
+                    (double)estimate / core::Millisecond, (double)lat_update_upper_thrsh_,
+                    (double)estimate * (double)lat_update_upper_thrsh_
+                        / core::Millisecond,
+                    (double)cur_tl_ns / core::Millisecond, (double)lat_update_dec_step_,
+                    (double)(new_tl_ns / core::Millisecond));
+
+            cur_tl_ns = new_tl_ns;
+            target_latency_ = new_tl_ts;
+            last_target_latency_update_ = now;
+            target_latency_state_ = TL_DEC_TIMEOUT;
+            fe_->update_target_latency((packet::stream_timestamp_t)target_latency_);
+
+            return;
+            // If evaluated target latency is greater, than we must increase it.
+        } else if (estimate > cur_tl_ns) {
+            const core::nanoseconds_t new_tl_ns =
+                (core::nanoseconds_t)(estimate * lat_update_inc_step_);
+            packet::stream_timestamp_diff_t new_tl_ts =
+                sample_spec_.ns_2_stream_timestamp_delta(new_tl_ns);
+
+            if (new_tl_ts > max_latency_) {
+                if (now > last_lat_limit_log_
+                    && (now - last_lat_limit_log_) > 5 * core::Second) {
+                    last_lat_limit_log_ = now;
+
+                    roc_log(LogDebug,
+                            "Latency tuner:"
+                            " capping target latency %ld(%.3fms)"
+                            " as max limit is lower %ld(%.3fms)",
+                            (long)new_tl_ts, (double)new_tl_ns / core::Millisecond,
+                            (long)max_latency_,
+                            sample_spec_.stream_timestamp_delta_2_ms(max_latency_));
+                }
+                new_tl_ts = max_latency_;
+            }
+
+            roc_log(LogInfo,
+                    "Latency tuner:"
+                    " increasing target latency %ld(%.3fms) → %ld(%.3fms)",
+                    (long)target_latency_, (double)cur_tl_ns / core::Millisecond,
+                    (long)new_tl_ts, (double)new_tl_ns / core::Millisecond);
+
+            target_latency_ = new_tl_ts;
+            last_target_latency_update_ = now;
+            target_latency_state_ = TL_INC_TIMEOUT;
+            fe_->update_target_latency((packet::stream_timestamp_t)target_latency_);
+
+            return;
+        }
+        // Waiting the timeout since last decreasement.
+    } else if (target_latency_state_ == TL_DEC_TIMEOUT
+               && now - last_target_latency_update_ > 15 * core::Second) {
+        target_latency_state_ = TL_NONE;
+        // Waiting the timeout since the startup.
+    } else if (target_latency_state_ == TL_START
```

Comment:
I guess clang-format made this comments weird. You could move them inside else if branch.